includes/documents/class-wcpdf-invoice.php

```diff
@@ -441,7 +441,7 @@ public function get_pdf_settings_fields( $option_name ) {
 				'args'			=> array(
 					'option_name'	=> $option_name,
 					'id'			=> 'invoice_number_search',
-					'description'   => __( 'Can potentially slow down the search process.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description'   => __( 'Can potentially slow down the search process. You can also use the <a href="https://woocommerce.com/document/high-performance-order-storage/" target="_blank">HPOS</a> feature to search orders by invoice numbers using search type selector.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
> Can potentially slow down the search process.

I think this only applies to non-HPOS stores.

Reply:
Maybe one that could combine the 2 situations? Suggestion:

> The search process may be slower on non-HPOS stores. For a more efficient search, you can utilize the <a href="https://woocommerce.com/document/high-performance-order-storage/" target="_blank">HPOS</a> feature, allowing you to search orders by invoice numbers using the search type selector.

Reply:
@alexmigf great suggestion I agree with it!